pypredict/requirements.txt

```
@@ -0,0 +1,2 @@
+fastapi
```

```suggestion
fastapi
jinja2
```

I needed to install this in my virtualenv to get the web server to run.